entry_points.txt

```diff
@@ -81,6 +81,7 @@ orchestration_templates = cfme.services.catalogs.orchestration_template:Orchestr
 physical_providers = cfme.physical.provider:PhysicalProviderCollection
 physical_servers = cfme.physical.physical_server:PhysicalServerCollection
 physical_switches = cfme.physical.physical_switch:PhysicalSwitchCollection
+planning = cfme.optimize.planning:PlanningCollection
```

Comment:
Collections should be plural names, and a bit more detailed. Consider `optimize_plans`